Specifications/catalog/catalog.binding.https.md

```diff
@@ -0,0 +1,134 @@
+# Catalog HTTPS Binding
+
+## 1 Introduction
+
+This specification defines a RESTful API over HTTPS for the [Catalog Protocol].
+
+The OpenAPI definitions for this specification can be accessed [here](TBD).
+
+## 2 Path Bindings
+
+### 2.1 Prerequisites
+
+1. The `<base>` notation indicates the base URL for a catalog service endpoint. For example, if the base catalog URL is `api.example.com`, the URL `https://<base>/catalog/request`
+   will map to `https//api.example.com/catalog/request`.
+
+2. All request and response messages must use the `application/json` media type.
```

Comment:
This opens the discussion of JSON vs. JSON-LD. IDS, DCAT, ODRL, Gaia-X, and many others relevant for us go with JSON-LD/RDF rather than plain (and therefor a lot simpler!) JSON.

Reply:
+1 for simple JSON!

IMO the only thing we should require of a runtime implementation is the ability to parse and write JSON (and, by extension, JSON-LD). There is no need to require any semantic processing in the specification. A simple runtime can be built which is capable of conveying semantic metadata without having to understand it. For example, A DCAT catalog serialized as JSON-LD could be deserialized and JSON-LD expansion applied to nodes so that namespaces are resolved and persisted along with asset attributes in a persistent store. A "semantic query" tool could be used to manipulate the persisted data if someone wanted to do that.